src/components/Pages/Join.tsx

```diff
@@ -89,38 +123,22 @@ const Join = () => {
           into &quot;circles&quot; by job function:
         </p>
       </div>
+
       <div
         style={{
           height: 'fit-content',
           display: 'grid',
           gridTemplateColumns: 'repeat(auto-fit, minmax(150px, 1fr))'
         }}
       >
-        <Circle
-          icon={<DevelopmentIcon />}
-          title="Development"
-          description="Coding and Programming"
-        />
-        <Circle
-          icon={<DataIcon />}
-          title="Data"
-          description="Collecting and organizing app data"
-        />
-        <Circle
-          icon={<ManagementIcon />}
-          title="Project Management"
-          description="Task management"
-        />
-        <Circle
-          icon={<DesignIcon />}
-          title="Design"
-          description="UX/UI Design and Resarch"
-        />
-        <Circle
-          icon={<CivicIcon />}
-          title="Civic"
-          description="Publicity and Community Engagement"
-        />
+        {ROLE_CARDS.map(card => (
```

Comment:
While this approach reduces repetition in the JSX, it makes the component's flow less intuitive, as you have to scroll up to see which circles we render. It also results in more lines of code. I'm not too concerned about mapping these circle components and would rather use this approach for dynamic data, where we don't know what data we receive, rather than for static data that doesn't change.